src/commands/mocks/azure-simple/terragrunt.hcl

```diff
@@ -0,0 +1,28 @@
+inputs = {
+    agent_vm_count = "3" 
+    agent_vm_size = "Standard_D2s_v3" 
+    acr_enabled = "true" 
+    gc_enabled = "true" 
+    cluster_name = "" 
+    dns_prefix = "" 
+    flux_recreate = "" 
+    kubeconfig_recreate = "" 
+    gitops_ssh_url = "" 
+    gitops_ssh_key = "" 
+    gitops_path = "" 
+    gitops_url_branch = "master" 
+    resource_group_name = "" 
+    ssh_public_key = "" 
+    service_principal_id = "" 
+    service_principal_secret = "" 
+    gitops_poll_interval = "5m" 
+    vnet_name = "" 
+    service_cidr = "10.0.0.0/16" 
+    dns_ip = "10.0.0.10" 
+    docker_cidr = "172.17.0.1/16" 
+    address_space = "10.10.0.0/16" 
+    subnet_prefix = "10.10.1.0/24" 
+    network_plugin = "azure" 
+    network_policy = "azure" 
+    oms_agent_enabled = "false"
+  }
```

Comment:
Let's remove this. This is carryover from another branch or PR.

Reply:
Removed.